.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,178 @@
+name: Prepare for a release
+
+on:
+  workflow_dispatch:
+    inputs:
+      component:
+        type: choice
+        options:
+        - OpenTelemetry.Exporter.Geneva
+        - OpenTelemetry.Exporter.InfluxDB
+        - OpenTelemetry.Exporter.Instana
+        - OpenTelemetry.Exporter.OneCollector
+        - OpenTelemetry.Exporter.Stackdriver
+        - OpenTelemetry.Extensions
+        - OpenTelemetry.Extensions.AWS
+        - OpenTelemetry.Extensions.Enrichment
+        - OpenTelemetry.Instrumentation.AspNet
+        - OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+        - OpenTelemetry.Instrumentation.AspNetCore
+        - OpenTelemetry.Instrumentation.AWS
+        - OpenTelemetry.Instrumentation.AWSLambda
+        - OpenTelemetry.Instrumentation.Cassandra
+        - OpenTelemetry.Instrumentation.ElasticsearchClient
+        - OpenTelemetry.Instrumentation.EntityFrameworkCore
+        - OpenTelemetry.Instrumentation.EventCounters
+        - OpenTelemetry.Instrumentation.GrpcCore
+        - OpenTelemetry.Instrumentation.GrpcNetClient
+        - OpenTelemetry.Instrumentation.Hangfire
+        - OpenTelemetry.Instrumentation.Http
+        - OpenTelemetry.Instrumentation.MassTransit
+        - OpenTelemetry.Instrumentation.MySqlData
+        - OpenTelemetry.Instrumentation.Owin
+        - OpenTelemetry.Instrumentation.Process
+        - OpenTelemetry.Instrumentation.Quartz
+        - OpenTelemetry.Instrumentation.Runtime
+        - OpenTelemetry.Instrumentation.SqlClient
+        - OpenTelemetry.Instrumentation.StackExchangeRedis
+        - OpenTelemetry.Instrumentation.Wcf
+        - OpenTelemetry.PersistentStorage.Abstractions
+        - OpenTelemetry.PersistentStorage.FileSystem
+        - OpenTelemetry.Resources.AWS
+        - OpenTelemetry.Resources.Azure
+        - OpenTelemetry.Resources.Container
+        - OpenTelemetry.Resources.Gcp
+        - OpenTelemetry.Resources.Host
+        - OpenTelemetry.Resources.Process
+        - OpenTelemetry.Resources.ProcessRuntime
```

Comment:
Just FYI I went with `OpenTelemetry.Resources` for all of these here. Seems like we'll be there soon. For ones that aren't updated, they can still be released doing everything manually the old way. Or we could just use the existing names and fix them up as we go. I thought this might save a bit of work.